freqtrade/freqtradebot.py

```diff
@@ -182,6 +185,13 @@ def _process(self, nb_assets: Optional[int] = 0) -> bool:
             final_list = sanitized_list[:nb_assets] if nb_assets else sanitized_list
             self.config['exchange']['pair_whitelist'] = final_list
 
+            # Calculating Edge positiong
+            # Should be called before refresh_tickers
+            # Otherwise it will override cached klines in exchange
+            # with delta value (klines only from last refresh_pairs)
+            if self.config['edge']['enabled']:
```

Comment:
use `config.get('edge', {}).get('enabled', None')` ... otehrwise it'll break with configurations which don't have thie configured... (this is not just here - but wherever this combination is used)